### ISSUE TITLE
list-item-indent: incorrect warnings for bullets which are not incremented

### ISSUE THREAD

**Issue by bachp** [wooorm/remark-lint#40]
If a ordered list of the following style is created:

```
1. One
1. Two
1. Three
1. Four
1. Five
1. Six
1. Seven
1. Eight
1. Nine
1. Ten
```

remark-lint complains for the last line:
`Incorrect list-item indent:add 1 space ...`

I assume the problem is that remark replaces `1. Ten` with `10. Ten`.

I don't think this is the intended behavior.

**Comment by wooorm** [comment]
Yes it is the intended behaviour, by default. See the settings for this rule on how to [turn it off or change the behaviour](https://github.com/wooorm/remark-lint/blob/master/doc/rules.md#list-item-indent).

**Comment by wooorm** [comment]
Also: are you sure this warning only occurs at the tenth item? I get warnings for every line. Could you show me your configuration?

**Comment by bachp** [comment]
The configuration I tried this with is:

```
  "remarkConfig": {
    "settings": {
        "connmark": true,
        "gfm": true
    },
    "plugins": {
      "lint": {
        "maximum-line-length": 200,
        "maximum-heading-length": 100,
        "list-item-indent": "space",
        "list-item-spacing": false,
        "ordered-list-marker-value": false,
        "no-heading-punctuation": ".!:",
        "no-duplicate-headings": false,
        "table-pipe-alignment": false,
        "table-cell-padding": false
      }
    }
  }
```

**Comment by wooorm** [comment]
Ai, my mistake. You’re definitely right.
